charts/drone/templates/serviceaccount.yaml

```
@@ -0,0 +1,9 @@
+{{- if .Values.serviceAccount.create }}
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ .Values.serviceAccount.name }}
+  annotations:
+{{ toYaml .Values.serviceAccount.annotations | indent 4 }}
```

I like using `nindent` to keep the template indentation aligned with the yaml indentation.
```suggestion
    {{- toYaml .Values.serviceAccount.annotations | nindent 4 }}
```